sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -20,7 +20,7 @@ import { PollerLike, PollOperationState } from "@azure/core-lro";
 import {
   DeletionRecoveryLevel,
   KnownDeletionRecoveryLevel,
-  KeyVaultClientGetKeysOptionalParams,
+  GetKeysOptionalParams,
```

Comment:
These changes just absorb the corresponding name changes in the TS codegen so we can use the latest and greatest